Description: Don't run the tests with JUnit 5 (not in Debian yet)
Author: Emmanuel Bourg <ebourg@apache.org>
Forwarded: not-needed
--- a/build.gradle
+++ b/build.gradle
@@ -57,7 +57,6 @@
   apply plugin: 'jacoco'
   sourceCompatibility = '1.8'
   targetCompatibility = '1.8'
-  test { useJUnitPlatform() }
   ext.provides = []  // The provided java packages, e.g. ['org.objectweb.asm']
   ext.requires = []  // The required Gradle projects, e.g. [':asm-test']
   ext.transitiveRequires = { ->
